cuegui/cuegui/FrameMonitorTree.py

```diff
@@ -410,14 +410,9 @@ def _getUpdateChanged(self):
         updatedFrames = job_pb2.UpdatedFrameSeq()
```

Comment:
Does this default value still work with the new code? I think this would want to revert to being a plain list instead of a FrameSeq?